### ISSUE TITLE
Pro License works for 2019.2.11f1 works but not 2018.4.10f 

### ISSUE THREAD

**Issue by thorgeirk11** [webbertakken/unity-actions#32]
@thorgeirk11 I have never seen a pro license but I would guess that is indeed what it would look like.

Could you try the same workflow with unity version 2019.2.11f1 to verify your credentials are correct? And if they are, we need to create a new issue for activating in 2018 versions, as I've never tested any.

_Originally posted by @webbertakken in https://github.com/webbertakken/unity-actions/issues/26#issuecomment-575521058_

**Comment by webbertakken** [comment]
> Furthermore since I was using email, password and serial I was under the impression that the unity-request-activation-file was not necessary.

Correct. The activation file is not needed for professional licenses. Just the three variables that you mentioned.

**Comment by webbertakken** [comment]
Could you try with https://github.com/webbertakken/unity-builder/releases/tag/v0.7 without using the license file (don't define `UNITY_LICENSE` at all), but just your serial, username and password?

For professional licenses it is not necessary to manually activate.

**Comment by thorgeirk11** [comment]
v0.7 works for the build action! However the return return-license action seems to be not working.

![image](https://user-images.githubusercontent.com/2693474/72730002-6425a080-3b88-11ea-8ded-0d85ee13c8d8.png)
